src/get-started/test-service.md

```diff
@@ -60,21 +60,30 @@ contain the tests for transaction business logic.
 ### Imports
 
 The created file is executed separately from the service code, meaning that we
```

Comment:
I believe this sentence makes little sense with the new code excerpt: in Rust-2018, it's no longer necessary to import crates in the source code (which the section previously illustrated), so it looks weird. I'd suggest to remove the sentence and the following code sample entirely.